crates/next-core/js/src/ipc/server.ts

```diff
@@ -106,11 +106,10 @@ export function makeRequest(
       port: address.port,
       method,
       path:
-        query && Object.keys(query).length > 0
-          ? `${path}?${qs.stringify(query)}`
-          : path,
-      headers,
+        rawQuery != null && rawQuery.length > 0 ? `${path}?${rawQuery}` : path,
+      headers: rawHeaders != null ? headersFromEntries(rawHeaders) : undefined,
```

Comment:
I've always wished we had an inverse `??` for chaining: `headers: rawHeaders !? headersFromEntries(rawHeaders)`.